### ISSUE TITLE
Bug: apparent disagreement on number of HDUs

### ISSUE THREAD

**Issue by weaverba137** [desihub/desidatamodel#170]
When verifying `DESI_SPECTRO_REDUX`, some files on disk appear to disagree with the data model on the number of HDUs. However when manually inspecting both the data file and the model file, the number of HDUs *does* agree. Track this down so that `DESI_SPECTRO_REDUX` can be fully verified.

**Comment by weaverba137** [comment]
Problem identified: some text in some data model files contains a certain phrase, *e.g.*:
```
See :doc:`coadd files <coadd-SURVEY-PROGRAM-PIXNUM>` for a coadded
```
However that is a signal to the data model software that the referenced document is the *actual* data model for the file, *i.e.* a cross reference.  We use this to cross reference the data model for fiberassign and fibermap files, for example, but in the example above, when `spectra-SURVEY-PROGRAM-PIXNUM.rst` is read, this is interpreted as a command to use `coadd-SURVEY-PROGRAM-PIXNUM.rst` as the data model instead.

@sbailey, please take note.

This can be solved with a simple rephrasing of the text, which will appear in #169.

**Comment by weaverba137** [comment]
I've also found cases where a cross-reference is *intended*, but it did *not* follow the phrasing above.

**Comment by weaverba137** [comment]
Closed by #169.
